code/modules/unit_tests/_unit_tests.dm

```diff
@@ -9,11 +9,23 @@
 
 /// Asserts that the two parameters passed are equal, fails otherwise
 /// Optionally allows an additional message in the case of a failure
-#define TEST_ASSERT_EQUAL(a, b, message) if ((a) != (b)) { return Fail("Expected [isnull(a) ? "null" : a] to be equal to [isnull(b) ? "null" : b].[message ? " [message]" : ""]") }
+#define TEST_ASSERT_EQUAL(a, b, message) do { \
```

Comment:
The `do { ... } while(FALSE)` here, for anyone unfamiliar, is a common pattern to get around duplicate assignment errors.